docs/en/sql-reference/functions/string-functions.md

```diff
@@ -88,20 +88,93 @@ Result:
 
 ## length
 
-Returns the length of a string in bytes (not: in characters or Unicode code points).
-
-The function also works for arrays.
+Returns the length of a string in bytes rather than in characters or Unicode code points. The function also works for arrays.
 
 Alias: `OCTET_LENGTH`
 
+**Syntax**
+
+```sql
+length(s)
+```
+
+**Parameters**
+
+- `s`: An input string. [String](../data-types/string)/[Array](../data-types/array).
```

Comment:
I think we should mention here that Array data type is also possible, as you said on l.91